src/backend/loader.js

```diff
@@ -23,14 +24,23 @@ const initialScope = sentryUserId ? { user: { id: sentryUserId } } : undefined
 
 /** @type {Array<"error" | "log" | "warn">} */
 const logLevels = ['error']
-
 let enableLogs = false
 
 if (sentryEnvironment !== 'production') {
   logLevels.push('log', 'warn')
   enableLogs = true
 }
 
+if (enableLogs) {
```

Comment:
Is `loader.js` running in the same bundle as the MapeoManager? If not we should do this update inside `src/backend/index.js`. Else it looks good to me,

Reply:
They are separate bundles, however rollup is also splitting the bundles for shared code like `debug`, so it's the same debug import being used by both the loader.js bundle and the app.js bundle. I verified this by checking that debug logging is still enabled in my dev build, and by also checking `debug.enabled()`.